### ISSUE TITLE
Install JUnit Dependencies

### ISSUE THREAD

**Issue by Pharaoh439** [Pharaoh439/capstone#8]

**Comment by Pharaoh439** [comment]
https://dzone.com/articles/unit-testing-w-junit-using-maven-and-intellij-pt1

**Comment by Pharaoh439** [comment]
Looks like we need to refactor the entire structure of our maven project. I'm going to start on this now, and it will probably break everything before it fixes anything.

**Comment by Pharaoh439** [comment]
This command got maven to generate a test directory as well as import the proper JUnit libraries.
mvn archetype:generate -DgroupId=in.patrickmart -DartifactId=engine -DarchetypeArtifactId=maven-archetype-quickstart -DinteractiveMode=false
I then merged the generated pom.xml with our old one to import LWJGL again. 
Neither the generated pom nor the old one actually created a runnable .jar, so I followed Andrew Thompson's suggestions [here](https://stackoverflow.com/questions/9689793/cant-execute-jar-file-no-main-manifest-attribute) to get a runnable application.
